neuralmonkey/decoders/output_projection.py

```diff
@@ -49,9 +49,8 @@ def _projection(prev_state, prev_output, ctx_tensors):
     return _projection
 
 
-# TODO fix this so it does not need dropout_plc. But it needs to be
```

Comment:
Tohle TODO tu musí zůstat, ten problém se neopravil. Jenom jsi dropout placeholder nahradil train_mode placeholderem. Musí se to vymyslet tak, aby tahle funkce nemusela dostávat žádný tenzory, protože se volá z konfiguráku a tam žádný nejsou.